content/23.vaccines-novel.md

```diff
@@ -130,14 +144,15 @@ Thus, in addition to lacking an infectious agent, these approaches are likely to
 ### DNA Vaccine Platforms
 
 DNA vaccine technologies have developed slowly over the past thirty years.
+DNA vaccines introduce a plasmid containing a DNA sequence that encodes the antigen(s) necessary to induce a specific immune response into appropriate tissues where _in situ_ expression of the target antigen is induced [@doi:10.1046/j.1365-2796.2003.01140.x].
```

Comment:
```suggestion
DNA vaccines introduce a vector containing a DNA sequence that encodes the antigen(s) necessary to induce a specific immune response into appropriate tissues where _in situ_ expression of the target antigen is induced [@doi:10.1046/j.1365-2796.2003.01140.x].
```
@RLordan I think it might be better to say vector since we discuss both plasmid- and viral-vectored vaccines, does that sound OK to you?

Reply:
Yes I think that should work fine.